modules/oe_theme_content_document_reference/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenEuropa Theme Document reference module
```

Comment:
The name of the module must follow the same convention we had so far, let's change it into:
```
oe_theme_content_sub_entity_document_reference
```